### DISCUSSION TITLE
somewhat ugly link preview image

### DISCUSSION THREAD

**Discussion by Thane5** [ggarra13/mrv2#301] [Ideas]
I just sent someone a link to the mrv2 repo on discord, and it showed this preview image:
![image](https://github.com/user-attachments/assets/f0a19dfb-a6f2-434b-88d4-5fac9edd5273)

Can that be changed? And wouldnt it make sense to use this newer screenshot instead?
https://github.com/ggarra13/mrv2/blob/main/docs/www/files/Main_UI.png

**Comment by ggarra13** [comment]
Done.
